src/pages/index.js

```diff
@@ -4,54 +4,59 @@ import {
   settings,
   resetValidation,
 } from "../scripts/validation.js";
+import Api from "../utils/Api.js";
 import logo from "../images/logo.svg";
 import avatar from "../images/avatar.jpg";
 import pencilIcon from "../images/pencil.svg";
 import plusIcon from "../images/plus.svg";
 import closeIcon from "../images/close.svg";
+import pencilIconLight from "../images/pencil-light.svg";
 
+// Set initial image sources
 document.querySelector(".header__logo").src = logo;
 document.querySelector(".profile__avatar").src = avatar;
```

Comment:
`document.querySelector(".profile__avatar")`

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine